source/index.html.slim

```diff
@@ -47,14 +48,14 @@ title: Home
           .taster__content
             .taster__summary
               markdown:
-                Build your own strict data types, constraints and coercions with dry-types.
+                Build your own data types, constraints and coercions with dry-types.
             .taster__example
               markdown:
                 ```ruby
                 module Types
-                  include Dry::Types.module
+                  include Dry::Types()
```

Comment:
should it be like that or `include Dry.Types`? Which one is less confusing? :)

Reply:
@flash-gordon that's how I write it, but I don't have any real preference when I think about it. I gues using dot would make it clearer that we're actually calling a method 🤔

Reply:
Yep, writing empty parens is uncommon for ruby, and if you skip them, you'll accidentally include a different module. When it happens, dry-types raises an error saying `'Import Dry.Types, not Dry::Types'`

Reply:
Ah that's a very good reason why dot notation is better. I'll change it here.